packages/@react-aria/spinbutton/src/useSpinButton.ts

```diff
@@ -128,7 +128,15 @@ export function useSpinButton(
   // This ensures that macOS VoiceOver announces it as "minus" even with other characters between the minus sign
   // and the number (e.g. currency symbol). Otherwise it announces nothing because it assumes the character is a hyphen.
   // In addition, replace the empty string with the word "Empty" so that iOS VoiceOver does not read "50%" for an empty field.
-  textValue = textValue === '' ? stringFormatter.format('Empty') : (textValue || `${value}`).replace('-', '\u2212');
+  textValue =
+    textValue === '' ?
+    stringFormatter.format('Empty') :
+    (textValue || `${value}`)
+    .trim()
+    .replace('-', '\u2212')
+    // Replace negative value formatted using currencySign: 'accounting'
+    // with a value that can be announced using a minus sign.
+    .replace(/^\((.+)\)$/, '\u2212$1');
```

Comment:
I can't quite remember, but is there any case where the currency is outside the parenthesis?
like AUD (50.00) or is it always (AUD 50.00) there's also symbol and name

Reply:
I checked through each of our locales. There are a number of cases where `currencySign: 'accounting'` does not render using parentheses, but in all the cases that use parentheses, the currency symbol is contained within the parentheses.

Reply:
yeah, if i recall, that's the case with the locales. if there was an issue, it'd be in the English locale but with a different currency and/or numbering system

Reply:
maybe try it with JPY? i remember that one having an accounting currencySign

Reply:
Also this is assuming anything in parenthesis is a negative number, but this hook isn't specific to number field. Maybe we should move this logic into number field instead.

We could also just use a different formatter for the aria value than what we use for the input value, eg without accounting mode enabled. Might be easier and more resilient than regex replacement.

Reply:
that's a much better idea I think, 👍🏻